.github/workflows/ci.yml

```diff
@@ -14,10 +14,10 @@ env:
   # allow overriding Maven command
   MAVEN: ./mvnw
   # maven.wagon.rto is in millis, defaults to 30m
-  MAVEN_OPTS: "-Xmx512M -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000"
-  MAVEN_INSTALL_OPTS: "-Xmx2G -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000"
+  MAVEN_OPTS: "-Xmx2G -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000"
```

Comment:
@electrum This would leave lesser space for containers in tests. IIRC it was intentional to have different heap sizes for installing trino vs everything else.